promgen/templates/promgen/exporter_form.html

```diff
@@ -34,6 +34,40 @@ <h1>Project: {{ project.name }}</h1>
         <table class="table">
           {{ form.as_table }}
         </table>
+    <div class="labels-formset" style="padding: 8px">
+        {{ labels_formset.management_form }}
+        <div class="panel panel-default">
+            <div class="panel-heading" role="tab" id="labels-formset-panel-header">
+                <a role="button" data-toggle="collapse" href="#labels-formset-panel" aria-expanded="false"
```

Comment:
<img width="763" alt="label form" src="https://user-images.githubusercontent.com/89725/150749551-12e7cec5-3e4b-405e-bc3c-69b9218414d8.png">
Thinking back and forth a bit about this. It's probably good to have it toggle visible like you have it, but I'm wondering if we may need to make small adjustments to the text descriptions

Reply:
What adjustments you want to make?

Reply:
The main issue for me is the scrape exporter code. First of all there is a [TODO](https://github.com/line/promgen/blob/master/promgen/views.py#L620) in the code.  Probably API `/rest/project/<slug>/scrape` shoult know nothing about exporter register form fields and should receive a POST with body like `{"host":"blackbox", "port":9115, "scheme": "https", "path": "/probe", "query": {"target": "prometheus.io"}}`. If so, then `ExporterScrape` class could build the POST body from register exporter form then query API and return API response. Is that OK?
Maybe it is better to add a field `is_parameter` to `ExporterLabel`? In that case user is no need any more to use a prefix `__param_` and all logic about prefix could be moved to prometheus `render_config` function.
Also I have a question about API `/rest/project/<slug>/scrape`. Why exporter's id is omitted? The url could be `/rest/project/<slug>/exporter/<slug>/scrape`. In that case all url logic could be encapsulated in an `Exporter` itself.

Reply:
The reason the scrape code is as is, is because you can use the `test` button before it has been saved to the database, so we need to be able to POST the data to a test input when we don't already have an exporter_id.